common/include/pcl/experimental/executor/omp_executor.hpp

```diff
@@ -39,51 +40,63 @@ struct omp_executor {
 
   std::size_t max_threads = 0;
 
-  omp_executor(): omp_executor(0) {}
+  omp_executor() : omp_executor(0) {}
 
-  omp_executor(std::size_t max_threads): max_threads(max_threads) {
+  omp_executor(std::size_t max_threads) : max_threads(max_threads)
+  {
 #ifdef _OPENMP
     set_max_threads(max_threads);
 #endif
   }
 
-  bool set_max_threads(std::size_t max_threads) {
+  bool
+  set_max_threads(std::size_t max_threads)
+  {
 #ifdef _OPENMP
-    if (!max_threads) this->max_threads = omp_get_max_threads();
-    else this->max_threads = max_threads;
+    if (!max_threads)
+      this->max_threads = omp_get_max_threads();
+    else
+      this->max_threads = max_threads;
```

Comment:
Didn't we have a triplet operator here before?

Reply:
Converted to the ternary operator